src/privateDogu.html

```diff
@@ -0,0 +1,618 @@
+<!-- FIXME currently not available at all, third party dogus will need to re-evaluated in the future regarding licenses -->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Licenses</title>
+  <link rel="stylesheet" href="/info/assets/css/ces.css">
+  <link rel="stylesheet" href="/info/styles.css">
+  <link rel="icon" type="image/x-icon" href="/info/assets/images/favicon.ico">
+  <script src="insertLicense.js"></script>
+</head>
+<body>
+<div>
+  <div class="navbar navbar-default navbar-fixed-top">
+    <div class="container-fluid">
+      <div class="navbar-header">
+        <button class="navbar-toggle collapsed" id="collapse-button" type="button">
+          <span class="sr-only">Toggle navigation</span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </button>
+        <a class="navbar-brand">
+          <img class="img-responsive" alt="Cloudogu" src="/info/assets/images/blib-white-30px.png">
+        </a>
+      </div>
+      <div class="collapse navbar-collapse" id="collapseExample">
+        <ul class="nav navbar-nav navbar-left">
+          <li><a href="/info/about" style="font-size: 18px;">About Cloudogu</a></li>
+          <li class="active"><a href="/info/dogulicenses">Dogu Licenses</a></li>
+          <li><a href="/info/licenses">3rd Party Licenses</a></li>
+          <li><a href="/info/privacyPolicy">Privacy Policy</a></li>
+          <li><a href="/info/termsAndConditions">Terms and Conditions</a></li>
+          <li><a href="/info/imprint">Imprint</a></li>
+          <li><a href="/info/writtenOffer">Written Offer</a></li>
+        </ul>
+      </div>
+    </div>
+  </div>
+</div>
+<br>
+<br>
+<br>
+<div class="text-center">
+  <img alt="" src="/info/assets/images/blib-blue-160px.png">
+  <br>
+  <br>
+  <h1 class="heading text-muted">Dogu Licenses</h1>
```

Comment:
This heading 1 suggests the same content as `doguLicenses.html`. How are they different? Do we need a different wording here?